stable/cosi-bucket-kit/templates/job-readiness.yaml

```diff
@@ -186,13 +184,13 @@ spec:
                 --dry-run=client -o yaml | kubectl apply -f -
               # Create a configmap with the name of the secret from above.
               kubectl create configmap {{ $cmName }} -n {{ $cmNamespace }} \
-                --from-file=values.yaml=<(cat <<EOF
-                persistence:
-                  imageChartStorage:
-                    type: s3
-                    s3:
-                      existingSecret: {{ .credentialsSecretName }}
-                EOF
+                --from-file=values.yaml=<(cat <<'EOF'
+              persistence:
```

Comment:
helm was adding a chomping character to trim the space by 2 so i just adjusted the indent.